util/test/tests/GL/GL_Draw_Zoo.py

```diff
@@ -39,3 +39,6 @@ def check_capture(self):
 
         self.check(pipe.IsStripRestartEnabled())
         self.check((pipe.GetStripRestartIndex() & ((1 << (ib.byteStride*8)) - 1)) == 0xffff)
+        
+        drawcalls = self.controller.GetDrawcalls()
+        self.check(len(drawcalls) == 24)
```

Comment:
This check works for now but is easily broken in future if a new unrelated drawcall is added to test something, and it's not clear exactly what this is testing for. You should instead test more directly for what you're looking for - a drawcall that has `DrawFlags.ClearDepthStencil` set.